sdk/python/feast/infra/offline_stores/contrib/postgres_offline_store/postgres.py

```diff
@@ -229,6 +237,59 @@ def pull_all_from_table_or_query(
             on_demand_feature_views=None,
         )
 
+    if "adbc_driver_postgresql" in sys.modules:
```

Comment:
could we instead add this to the config optionally?

Reply:
What config?